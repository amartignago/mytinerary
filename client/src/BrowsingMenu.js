import React, {Component} from 'react';
import circledRight from './circled-right-2.png';
import './App.css';
import {
  Link
} from "react-router-dom";

class BrowsingMenu extends Component {
  render() { return ( 
      <div>
        <h1>Start Browsing</h1>
        <div>
          <img src={circledRight} className="browsing-icon" alt="browsing logo"></img>
          <div></div>
          <Link to="/cities">Cities</Link>
        </div>
        <div>
          <span>Want to build your own Mytinerary?</span>
        </div>
        <div className="float-left">
          <Link to="/login">Log in</Link>
        </div>
        <div className="float-right">
          <Link to="/Account">Account</Link>
        </div>
      </div>

  );
}
}

export default BrowsingMenu;
