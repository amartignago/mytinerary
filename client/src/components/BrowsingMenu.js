import React, {Component} from 'react';
import circledRight from '../images/circled-right-2.png';
import '../styles/App.css';
import LoginButton from './LoginButton'
import {
  Link
} from "react-router-dom";

class BrowsingMenu extends Component {
  render() { return ( 
      <div className="text-center">
        <h1>Start Browsing</h1>
        <div className="container responsiveDiv">
          <Link to="/cities">
            <img src={circledRight} className="browsing-icon" alt="browsing logo"></img>
          </Link>
        </div>
        <div>
          <span>Want to build your own Mytinerary?</span>
        </div>
        <div className="d-flex justify-content-around">
          <LoginButton></LoginButton>
          <Link to="/Account">Account</Link> 
          {/* crear componente para account button */}
        </div>
      </div>

  );
}
}

export default BrowsingMenu;
