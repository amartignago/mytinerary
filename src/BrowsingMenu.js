import React, {Component} from 'react';
import circledRight from './circled-right-2.png';
//import './App.css';

class BrowsingMenu extends Component {
  render() { return ( <div>
      <div>
        <h1>Start Browsing</h1>
        <div>
          <img src={circledRight} alt="browsing logo"></img>
        </div>
        <div>
          <span>Want to build your own Mytinerary?</span>
        </div>
        <div className="float-left">
          <a href="#">Log in</a>
        </div>
        <div className="float-right">
          <a href="#">Create Account</a>
        </div>
      </div>
    </div>
  );
}
}

export default BrowsingMenu;
