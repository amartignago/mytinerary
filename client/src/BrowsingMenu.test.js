import React from 'react';
import { shallow , mount} from 'enzyme';
import BrowsingMenu from './BrowsingMenu.js';


describe('BrowsingMenu', () => {
  it('should render correctly', () => {
    const component = shallow(<BrowsingMenu/>); 
    expect(component).toMatchSnapshot();
  });
});

