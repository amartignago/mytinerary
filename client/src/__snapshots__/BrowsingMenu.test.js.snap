// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrowsingMenu should render correctly 1`] = `ShallowWrapper {}`;
